### PR TITLE
[Filebeat][Salesforce] Add Salesforce dashboard screenshots to documentation

### PR DESCRIPTION
**Pull request by kush-elastic**
- Enhancement

## What does this PR do?

Adds a screenshot of a dashboard to the documentation of the Salesforce filebeat module.

## Why is it important?

We had a discussion with @akshay-saraswat and it is important to have it in the documentation to pull it for press release.

## Checklist

~~- [ ] My code follows the style guidelines of this project~~
~~- [ ] I have commented my code, particularly in hard-to-understand areas~~

- [X] I have made corresponding changes to the documentation
- [X] I have made corresponding change to the default configuration files

~~- [ ] I have added tests that prove my fix is effective or that my feature works~~
~~- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.~~

## Related issues

- Relates #31486

## Screenshots
filebeat-salesforce-login-dashboard
filebeat-salesforce-logout-dashboard

### CODE REVIEW

File: x-pack/filebeat/module/salesforce/_meta/docs.asciidoc
Comment (dedemorton) [inline x-pack/filebeat/module/salesforce/_meta/docs.asciidoc:296]:
```suggestion
This Salesforce module comes with several predefined dashboards, including Login, Logout, Apex, and Setup Audit Trails Dashboards. For example:
```

File: filebeat/docs/modules/salesforce.asciidoc
Comment (dedemorton) [inline filebeat/docs/modules/salesforce.asciidoc:303]:
```suggestion
This Salesforce module comes with several predefined dashboards, including Login, Logout, Apex, and Setup Audit Trails Dashboards. For example:
```

**Review by dedemorton — COMMENTED** [review]
A couple minor edits. Why is there no doc preview for this PR?

**Comment by kush-elastic** [comment]
> Why is there no doc preview for this PR?

I am not sure what you mean by `doc preview`. is there something I have do?
if there is anything missing please let me know.

**Comment by dedemorton** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by dedemorton** [comment]
@kush-elastic The doc build should have run as part of the CI tests, but didn't. So I have submitted a command to run it.

**Comment by bmorelli25** [comment]
> Why is there no doc preview for this PR?

The PR author isn't a member of the Elastic GitHub org, so for security reasons the test has to be kicked off manually.

**Comment by kush-elastic** [comment]
> @kush-elastic The doc build should have run as part of the CI tests, but didn't. So I have submitted a command to run it.

@dedemorton, thank you for the help.
if it seems good now, can we merge this PR?

**Review by dedemorton — APPROVED** [review]
LGTM. I assume we want to backport this to 8.2 and later, so I've added the backport tags. Thanks!

**Comment by dedemorton** [comment]
@ishleenk17 @lalit-satapathy My apologies. I should have waited for a review from someone on your team before merging this PR because it has changes to the manifest files. I haven't merged the backports yet, though. Can you take a look?

**Pull request merged**
